src/operator/nn/cudnn/cudnn_pooling-inl.h

```diff
@@ -125,7 +126,8 @@ class CuDNNPoolingOp {
     CHECK_EQ(s->dnn_handle_ownership_, mshadow::Stream<gpu>::OwnHandle);
     typename DataType<DType>::ScaleType alpha = 1.0f;
     typename DataType<DType>::ScaleType beta = 0.0f;
-    this->Init(s, in_data, out_data);
+    if (!this->Init(s, in_data, out_data))
+      LOG(FATAL) << "CuDNN Pooling invoked with unsupported parameters.";
```

Comment:
nit: is the official name cuDNN or CuDNN?

Reply:
Official name is cuDNN, even at the beginning of a sentence apparently.  Fixed.